app/src/scenes/Persons/PersonSummary.js

```diff
@@ -118,7 +118,8 @@ const PersonSummary = ({
       {person.outOfActiveList && (
         <AlterOutOfActiveList>
           <Text style={{ color: colors.app.colorWhite }}>
-            {person?.name} est en dehors de la file active, pour le motif suivant : {person.outOfActiveListReason}
+            {person?.name} est en dehors de la file active, pour{' '}
+            {person.outOfActiveListReasons.length > 1 ? 'les motifs suivants' : 'le motif suivant'} : {person.outOfActiveListReasons.join(', ')}
```

Comment:
```suggestion
            {person?.outOfActiveListReasons?.length > 1 ? 'les motifs suivants' : 'le motif suivant'} : {person.outOfActiveListReasons.join(', ')}
```

Pour être cohérent avec le reste